src/components/furniture/Separators.vue

```diff
@@ -95,29 +95,23 @@ export default {
       };
     },
     widthVariationRelatedToTypeOfSeparator() {
-      if (this.insideSpaces.typeOfSeparators === 'centered') {
-        return 0;
-      } else if (this.insideSpaces.typeOfSeparators === 'left') {
-        return -12 / this.insideSpaces.amountOfSeparators;
-      } else {
-        return 12 / this.insideSpaces.amountOfSeparators;
-      }
+      const objectTypeSeparators = {
```

Comment:
This whole component is pretty hard to understand and complex, let's take some time and refactor it together